sulis_module/src/item.rs

```diff
@@ -132,18 +132,7 @@ impl Item {
             &all_adjectives,
         );
 
-        let mut name = String::new();
```

Comment:
Removing the precompute of the name and computing it every time `get_name()` is called seems like it could be a pretty big performance hit.  Keep in mind this method could potentially be called hundreds (thousands?) of times on every UI state change.

I'd suggest keeping the name here and maybe also storing an unidentified name variant.

Reply:
ok that option crossed my mind as well, will do that.